package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "masys-smart-schedular",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
add linting command that automatically fixes the codebase like  "lint:fix": "next lint --fix"